modules/cnf-image-based-upgrade-share-container-directory.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+// * edge_computing/image-based-upgrade/cnf-preparing-for-image-based-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cnf-image-based-upgrade-shared-container-directory_{context}"]
+= Configuring a shared container directory between `ostree` stateroots
```

Comment:
Remove backticks in headings (per [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata)):
```suggestion
= Configuring a shared container directory between ostree stateroots
```